statusreports/REPORT_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# [Month Day Year] Week in Review 
```

Comment:
We should probably specify the format for "Month" (numeric or month name [clearer for those used to `DD/MM/YYYY` formats]), but it would probably be better to specify the date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format:

```
# [ISO 8601 data] Week in Review 
```

Or should this be a date range for the week in question? Or maybe just a 4 digit year plus [ISO week number](https://en.wikipedia.org/wiki/ISO_week_date)?

Reply:
@jodh-intel Opted for YYYY-MM-DD. Will change past status reports posted on the website to match.